### PR TITLE
Md backwards compatibility

### PR DESCRIPTION
**Pull request by liamhuber**
Fixing a bug where forced application of the old `delta_press` didn't work.

Also added rescaling of pressure units to accommodate non-metal Lammps units, and some minor documentation/beautification.

### CODE REVIEW

File: pyiron/lammps/control.py [hunk 1]
```diff
@@ -147,49 +147,66 @@ def calc_md(self, temperature=None, pressure=None, n_ionic_steps=1000, time_step
...
-                                  'real': 1,
-                                  'electron': 1}
-
+        # Conversion factors for transfroming pyiron units to Lammps units
+        fs_to_ps = 1.e-3
+        fs_to_s = 1.e-12
+        GPa_to_bar = 1.e4
+        GPa_to_Pa = 1.e9
+        GPa_to_barye = 1.e10
+        GPa_to_atm = 9869.232667160128  # digits c.f. http://www.kylesconverter.com/pressure/gigapascals-to-atmospheres
```
Comment (jan-janssen) [inline pyiron/lammps/control.py]: Can not we do the unit conversion with scipy constants?

Comment (liamhuber) [reply pyiron/lammps/control.py]: Certainly. I see to do it like this:

`import scipy.constants as sconst

fs_to_ps = sconst.femto / sconst.pico
fs_to_s = sconst.femto / 1.
GPa_to_bar = sconst.giga * 1. / sconst.bar
GPa_to_Pa = sconst.giga 
GPa_to_barye = sconst.giga * 1. / (1.e-6 * sconst.bar)  # Lammps is in "barye"
GPa_to_atm = sconst.giga * 1. / sconst.atm`

...but to me this is quite ugly looking compared to just putting the number. Is there a more elegant way to phrase it?

Comment (jan-janssen) [reply pyiron/lammps/control.py]: I am not aware if there is a more elegant way - so I guess for now we can just implement it like this.

Comment (liamhuber) [reply pyiron/lammps/control.py]: Amended.

**Pull request merged**